kuma/wiki/jinja2/wiki/document.html

```diff
@@ -202,7 +201,7 @@ <h1>{{ document.title }}</h1>
                 {% include 'wiki/includes/kumascript_errors.html' %}
               {% endif %}
 
-              {% if document.current_revision %}
+              {% if document.parent_id and document.current_revision %}
```

Comment:
This is probably a good optimization. I'll need to check current docs to see if skips the section when it shouldn't.